src/test/resources/TestNG.xml

```diff
@@ -6,7 +6,12 @@
     <test name="Example Test" preserve-order="false">
         <classes>
             <class name="LoginTests"/>
+            <class name="ProfileTests"/>
             <class name="BaseTest"/>
+            <class name="Homework17"/>
+            <class name="Homework16"/>
+            <class name="Homework18"/>
```

Comment:
//comment from hw team: Homework requirement 18: XML File Configurations requires you to Use the <methods> & <include> tags in the XML file, but currently I have not seen you use it.

TestNg provides an option to include or exclude Groups, Test Methods, Classes and Packages using include and exclude tags by defining in testng.xml.
You will create a Class with Test Methods. and include test methods into xml file. in your Homewok18 you already have the playSong method, so you need to include that method into xml file

//HINT:
![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/144403954/78dd5e78-aa3a-4351-8d22-12164b5a90dd)